acceleration/transform_speed.ipynb

```diff
@@ -146,7 +138,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "/workspace/data/medical\n"
+      "/home/rbrown/data/MONAI\n"
```

Comment:
Please don't use your own path. I didn't check all the path in this PR, could you please help double confirm whether you also changed the path in other places?
Thanks.

Reply:
solved throughout, thanks